python/cudf/bindings/dlpack.pyx

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+from cudf.bindings.cudf_cpp cimport *
+from cudf.bindings.cudf_cpp import *
+from librmm_cffi import librmm as rmm
+
+from libc.stdint cimport uintptr_t, int8_t
+from libc.stdlib cimport calloc, malloc, free
+from cpython cimport pycapsule
+
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+
+
+cpdef from_dlpack(dlpack_capsule):
+    """
+    Converts a DLPack Tensor PyCapsule into a list of cudf Column objects.
```

Comment:
Suggest documenting what "name" and data type are expected inside the DLPack capsule.